policy/modules/system/systemd.if

```diff
@@ -322,7 +322,7 @@ interface(`systemd_write_notify_socket',`
 
 	init_list_runtime($1)
 	init_unix_stream_socket_sendto($1)
-	allow $1 systemd_runtime_notify_t:sock_file write;
+	allow $1 systemd_runtime_notify_t:sock_file { getattr write };
```

Comment:
Please use `write_sock_file_perms` instead.

Reply:
This would include `open`, is that desired?

Reply:
I don't have a concern here.

Reply:
Adjusted.